homeassistant/components/acer_projector/switch.py

```diff
@@ -87,7 +105,7 @@ def __init__(self, serial_port, name, timeout, write_timeout, **kwargs):
             ECO_MODE: STATE_UNKNOWN,
         }
 
-    def _write_read(self, msg):
+    def _write_read(self, msg: str) -> str | None:
```

Comment:
Or probably it just can't return `None`.
```suggestion
    def _write_read(self, msg: str) -> str:
```